docs/sql-ref-syntax-qry-select-where.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: global
+title: SELECT
+displayTitle: WHERE clause
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+The <code>WHERE</code> clause is used to limit the results of the <code>FROM</code>
+clause of query or subquery based on the specified condition.
+
+### Syntax
+{% highlight sql %}
+WHERE boolean_expression
+{% endhighlight %}
+
+### Parameters
+<dl>
+  <dt><code><em>boolean_expression</em></code></dt>
+  <dd>
+    Specifies any expression that evaluates to a result type <code>boolean</code>. Two or
+    more expressions may be combined together using comparision operators 
+    ( <code>></code>, <code>>=</code>, <code>=</code>, <code><</code>, <code><=</code>
```

Comment:
You can't combine boolean expressions with `<`, etc. though, which sounds like what this implies. You just want to mention the boolean operators like `AND`, right?

Reply:
@srowen I think you are right Sean. On the second read, i agree. It sounds confusing.  I will drop the comparison operators from this.